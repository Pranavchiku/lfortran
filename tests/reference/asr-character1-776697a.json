{
    "basename": "asr-character1-776697a",
    "cmd": "lfortran --indent --show-asr --allow-implicit-interface --no-color {infile} -o {outfile}",
    "infile": "tests/character1.f90",
    "infile_hash": "e488ad1d203df5884bda7425ec44e96927695edf53e10b7f55004b50",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-character1-776697a.stdout",
    "stdout_hash": "a08d16756f2454b52e2e15a708d11414aa8effaf4cfda3872fec0cdc",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}