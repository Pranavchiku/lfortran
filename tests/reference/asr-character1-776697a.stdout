(TranslationUnit
    (SymbolTable
        1
        {
            xerror:
                (Function
                    (SymbolTable
                        2
                        {
                            mess:
                                (Variable
                                    2
                                    mess
                                    []
                                    Unspecified
                                    ()
                                    ()
                                    Default
                                    (Character 1 1 () [])
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    xerror
                    []
                    [(Var 2 mess)]
                    [(Print
                        ()
                        [(ArrayItem
                            (Var 2 mess)
                            [(()
                            (IntegerConstant 1 (Integer 4 []))
                            ())]
                            (Character 1 1 () [])
                            ColMajor
                            ()
                        )]
                        ()
                        ()
                    )
                    (Return)]
                    ()
                    Source
                    Public
                    Implementation
                    ()
                    .false.
                    .false.
                    .false.
                    .false.
                    .false.
                    []
                    []
                    .false.
                )
        })
    []
)
